src/overlays/actors/ovl_Bg_Spot15_Rrbox/z_bg_spot15_rrbox.c

```diff
@@ -9,7 +9,20 @@ void BgSpot15Rrbox_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot15Rrbox_Update(Actor* thisx, GlobalContext* globalCtx);
 void BgSpot15Rrbox_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
+void func_808B4084(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B40AC(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B4194(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B4380(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B43D0(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B44B8(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B4178(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+void func_808B44CC(BgSpot15Rrbox* this, GlobalContext* globalCtx);
+
+s16 D_808B4590 = 0;
+
+extern s32 D_06000348;
```

Comment:
use `UNK_TYPE` (just a define resolving to s32, but indicates the type is... unknown) for `D_06000348`, it's the mesh header, we're waiting for z_bgcheck to have a proper struct for it
`func_808B3960` takes a pointer to this, also change it to a UNK_TYPE pointer